src/AutoPilotPlugins/PX4/SensorsComponentSummaryFixedWing.qml

```diff
@@ -25,7 +25,7 @@ FactPanel {
     property Fact airspeedDisabledFact: controller.getParameterFact(-1, "FW_ARSP_MODE")
     property Fact airspeedBreakerFact:  controller.getParameterFact(-1, "CBRK_AIRSPD_CHK")
 
-    property bool _airspeedVisible:     airspeedDisabledFact.value === false && airspeedBreakerFact.value !== 162128
+    property bool _airspeedVisible:     airspeedDisabledFact.value == 0 && airspeedBreakerFact.value !== 162128
```

Comment:
airspeedDisabledFact -> airspeedModeFact?

Reply:
Better, but not super critical